train.py

```diff
@@ -143,7 +173,7 @@
 clip_value = 1e-2
 config["clip_value"] = clip_value
 for p in model.parameters():
-    if p.requires_grad:
+    if p.requires_grad and config['competition'] == 'south_africa':
```

Comment:
What's going on here?

Reply:
> this clipping is needed for Germany dataset
Right now this code disables clipping for the Germany dataset, unless I am misunderstanding something

> loss is nan
Do you know why this is?

Reply:
I think this is the only thing that needs to be resolved!

Reply:
Any update on this?

Reply:
I've been working on getting the LSTM implementation first, so I'll get back to it once it's over.